src/ocm/components/clusterConfiguration/networkConfiguration/VirtualIPControlGroup.tsx

```diff
@@ -0,0 +1,193 @@
+          ? validation.message
+          : undefined;
+    }
+    return lookup;
+  }, {});
+};
+
+export type VirtualIPControlGroupProps = {
+  cluster: Cluster;
+  hostSubnets: HostSubnets;
+  isVipDhcpAllocationDisabled?: boolean;
+};
+
+export const VirtualIPControlGroup = ({
+  cluster,
+  hostSubnets,
+  isVipDhcpAllocationDisabled,
+}: VirtualIPControlGroupProps) => {
+  const { values, setFieldValue } = useFormikContext<NetworkConfigurationValues>();
+
+  const apiVipHelperText = `Provide an endpoint for users, both human and machine, to interact with and configure the platform. If needed, contact your IT manager for more information. ${getVipHelperSuffix(
+    cluster.apiVip,
+    cluster.vipDhcpAllocation,
+    values.vipDhcpAllocation,
+  )}`;
+  const ingressVipHelperText = `Provide an endpoint for application traffic flowing in from outside the cluster. If needed, contact your IT manager for more information. ${getVipHelperSuffix(
+    cluster.ingressVip,
+    cluster.vipDhcpAllocation,
+    values.vipDhcpAllocation,
+  )}`;
+
+  const {
+    'api-vip-defined': apiVipFailedValidationMessage,
+    'ingress-vip-defined': ingressVipFailedValidationMessage,
+  } = React.useMemo(() => getVipValidationsById(cluster.validationsInfo), [
+    cluster.validationsInfo,
+  ]);
+
+  const enableAllocation = useMemo(() => values.networkType === 'OpenShiftSDN', [values]);
+
+  useEffect(() => {
+    if (!enableAllocation) {
+      setFieldValue('vipDhcpAllocation', false);
+    }
+  }, [enableAllocation, setFieldValue]);
+
+  return (
+    <>
+      {!isVipDhcpAllocationDisabled && (
+        <CheckboxField
+          label={
+            <>
+              <Tooltip
+                hidden={enableAllocation}
+                content={
+                  "A cluster with OVN networking type cannot use 'allocate virtual IPs via DHCP server'"
+                }
+              >
+                <span>Allocate virtual IPs via DHCP server</span>
+              </Tooltip>
+              <FeatureSupportLevelBadge
+                featureId="VIP_AUTO_ALLOC"
+                openshiftVersion={cluster.openshiftVersion}
+              ></FeatureSupportLevelBadge>
+            </>
+          }
+          name="vipDhcpAllocation"
+          isDisabled={!enableAllocation}
+        />
+      )}
+      {values.vipDhcpAllocation ? (
+        <>
+          <FormikStaticField
+            label="API Virtual IP"
+            name="apiVip"
+            helperText={apiVipHelperText}
+            value={cluster.apiVip || ''}
+            isValid={!apiVipFailedValidationMessage}
+            isRequired
+          >
+            <VipStaticValue
+              vipName="apiVip"
+              cluster={cluster}
+              validationErrorMessage={apiVipFailedValidationMessage}
+            />
+          </FormikStaticField>
+          <FormikStaticField
+            label="Ingress Virtual IP"
```

Comment:
I have two PRs in which I removed references to "Virtual IP" and instead just refer to "IP".
Since this is a new file, it probably won't show any conflicts, but if my issues get merged first, then this would be a regression. We can keep an eye on what gets merged first to solve this

https://github.com/openshift-assisted/assisted-ui-lib/pull/1274
https://github.com/openshift-assisted/assisted-ui-lib/pull/1263
https://github.com/openshift-assisted/assisted-ui-lib/pull/1259

Reply:
@jkilzi please merge Celia's PRs regarding virtual ip -> ip after dual stack is merged

Reply:
ack

Reply:
@batzionb  @jkilzi Since these are new files, I will need to also update the PR to changes these new labels. I'll update Jira to keep it in mind.